hnn_core/gui/_viz_manager.py

```diff
@@ -68,6 +68,24 @@
 }
 
 
+def _align_fig_xaxis(fig):
+    # align x axis for all plots whose x axis are time.
+    ax_align_x = []
+    for ax in fig.get_axes():
+        if hasattr(ax, 'align_x'):
+            if getattr(ax, 'align_x'):
+                ax_align_x.append(ax)
+
+    if len(ax_align_x) > 1:
+        ax_align_x[0].get_shared_x_axes().join(*ax_align_x)
```

Comment:
To make sure I understand, this queries the xlimits from the first axis in the figure?

Reply:
yes